g2p/mappings/langs/kwk/napa_equiv_ubc.csv

```diff
@@ -1,70 +1,70 @@
-p/,p\u0313
-p\u0315,p\u0313
-p\u2019,p\u0313
-t/,t\u0313
-t\u0315,t\u0313
-t\u2019,t\u0313
-k/,k\u0313
-k\u0315,k\u0313
-k\u2019,k\u0313
-c/,c\u0313
-c\u0315,c\u0313
-c\u2019,c\u0313
-ƛ/,ƛ\u0313
-ƛ\u0315,ƛ\u0313
-ƛ\u2019,ƛ\u0313
-k{1}'{2}ʷ{3},k{1}\u0313{2}ʷ{3}
-k{1}]{2}ʷ{3},k{1}\u0313{2}ʷ{3}
-k{1}\u0315{2}ʷ{3},k{1}\u0313{2}ʷ{3}
-k{1}\u2019{2}ʷ{3},k{1}\u0313{2}ʷ{3}
+p/,p̓
+p̕,p̓
+p’,p̓
+t/,t̓
+t̕,t̓
+t’,t̓
+k/,k̓
+k̕,k̓
+k’,k̓
+c/,c̓
+c̕,c̓
+c’,c̓
+ƛ/,ƛ̓
+ƛ̕,ƛ̓
+ƛ’,ƛ̓
+k'ʷ,k̓ʷ
+k]ʷ,k̓ʷ
+k̕ʷ,k̓ʷ
+k’ʷ,k̓ʷ
 k{1}ʷ{3}\u0313{2},k{1}\u0313{2}ʷ{3}
 k{1}ʷ{3}'{2},k{1}\u0313{2}ʷ{3}
-k{1}ʷ{3}\u2019{2},k{1}\u0313{2}ʷ{3}
-k{1}'{2}w{3},k{1}\u0313{2}ʷ{3}
-k{1}]{2}w{3},k{1}\u0313{2}ʷ{3}
-k{1}\u0315{2}w{3},k{1}\u0313{2}ʷ{3}
-k{1}\u2019{2}w{3},k{1}\u0313{2}ʷ{3}
+k{1}ʷ{3}’{2},k{1}\u0313{2}ʷ{3}
+k'w,k̓ʷ
```

Comment:
there are lot of similar rules here collapsing the difference between ', \u2019 and ] - can these not be simplified?

Reply:
Yeah, here I didn't try to refactor anything, it's just a rewrite as is of someone's mapping. You're right, it can probably by simplified quite a bit. I have frequent similar gripes about many mappings in g2p, but I'm often scared of trying to fix them because I would really want to make sure the fix doesn't break anything else.